clang/include/clang/Basic/LangOptions.def

```diff
@@ -277,6 +277,7 @@ LANGOPT(SYCLIsDevice      , 1, 0, "Generate code for SYCL device")
 LANGOPT(SYCLIsHost        , 1, 0, "SYCL host compilation")
 LANGOPT(IntelFPGA         , 1, 0, "Perform ahead-of-time compilation for FPGA")
 LANGOPT(SYCLAllowFuncPtr  , 1, 0, "Allow function pointers in SYCL device code")
+LANGOPT(SYCLAllowMutableGlobal  , 1, 0, "Allow mutable global variables in SYCL device code")
```

Comment:
Do they exist also on the host these variables?

Reply:
no, This only allows variable on device to be mutable. host variables and device variable are still distinct.